gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/store/GemFireStore.java

```diff
@@ -1473,7 +1473,11 @@ private void bootRegions() throws StandardException {
               dsf.setMaxOplogSize(DiskStoreFactory.DEFAULT_MAX_OPLOG_SIZE);
             }
             else {
-              dsf.setMaxOplogSize(10);
+              if (this.myKind.isAccessor()) {
+                dsf.setMaxOplogSize(1);
+              } else {
+                dsf.setMaxOplogSize(10);
+              }
```

Comment:
Does the lead now creates only datadictionary diskstore, or default diskstore or both? I don't think it should be creating datadictionary diskstore to avoid confusion rather only the default diskstore with a reduced size as above. The flag "persistingDD" will govern whether or not datadictionary will be persistent so avoid touching that flag for lead and instead set the "persistenceDir" value to something for lead (or a special additional ORed check for lead in the block above).